operator/tasks/allocator_azure/allocator_azure.go

```diff
@@ -12,22 +12,33 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+package allocatorazure
```

Comment:
Same here, I'd suggest to make package name and direcory name consistent (depending on whether this is moved to `pkg/opam/allocator` as @aanm suggested.